classes/privacy/provider.php

```diff
@@ -0,0 +1,47 @@
+<?php
+// This file is part of the ocis repository for Moodle - http://moodle.org/
+//
+// The ocis repository for Moodle is free software: you can redistribute it and/or modify
+// it under the terms of the GNU General Public License as published by
+// the Free Software Foundation, either version 3 of the License, or
+// (at your option) any later version.
+//
+// The ocis repository for Moodle is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+// GNU General Public License for more details.
+//
+// You should have received a copy of the GNU General Public License
+// along with the ocis repository for Moodle.  If not, see <http://www.gnu.org/licenses/>.
+//
+
+
+/**
+ * Privacy provider.
+ *
+ * @package    repository_ocis
+ * @copyright  2018 Nina Herrmann (Learnweb, University of Münster)
```

Comment:
I do not think this is needed. Just leave the owncloud line.

Reply:
I thought its nice and fair to give credit to the person who [created the code and from where I've copied](https://github.com/moodle/moodle/blob/main/repository/nextcloud/classes/privacy/provider.php) it, same for other files:

Reply:
I also asked this in the dev chat. If it's just a copy of a file the copyright has to stay to the original copyright. But you can add an author tag.